Nodejs/Product/Nodejs/Options/NodejsIntellisenseOptionsControl.resx

```diff
@@ -520,10 +298,10 @@
     <value>GrowAndShrink</value>
   </data>
   <data name="$this.Margin" type="System.Windows.Forms.Padding, System.Windows.Forms">
-    <value>6, 6, 0, 6</value>
+    <value>3, 3, 0, 3</value>
   </data>
   <data name="$this.Size" type="System.Drawing.Size, System.Drawing">
-    <value>735, 745</value>
+    <value>368, 260</value>
```

Comment:
it's a little hard to visualize these changes - do you have a screenshot of the new UI?